lib/countly.js

```diff
@@ -627,6 +628,26 @@ Countly.Bulk = Bulk;
         return Countly.device_id;
     };
 
+    /**
+    * Changes the current device ID according to the device ID type (the preffered method)
+    * @param {string} newId - new user/device ID to use. Must be a non-empty string value. Invalid values (like null, empty string or undefined) will be rejected
+    * */
+    Countly.set_id = function(newId) {
+        cc.log(cc.logLevelEnums.INFO, `set_id, Changing the device ID to: [${newId}]`);
```

Comment:
lets add type check here also

Reply:
which type check? should we move it to the upper